<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="crypt.impl.certificate.X509V3GeneratorTest" tests="8" skipped="0" failures="0" errors="0" timestamp="2017-11-15T13:31:18" hostname="raphael-laptop" time="0.946">
  <properties/>
  <testcase name="testGetKeysPairException" classname="crypt.impl.certificate.X509V3GeneratorTest" time="0.018"/>
  <testcase name="testStoreInKeystore" classname="crypt.impl.certificate.X509V3GeneratorTest" time="0.673"/>
  <testcase name="testCreateChainCertificate" classname="crypt.impl.certificate.X509V3GeneratorTest" time="0.138"/>
  <testcase name="testCreateCertificateException" classname="crypt.impl.certificate.X509V3GeneratorTest" time="0.002"/>
  <testcase name="testCreateCertificateAndGetters" classname="crypt.impl.certificate.X509V3GeneratorTest" time="0.089"/>
  <testcase name="testBadInitFile" classname="crypt.impl.certificate.X509V3GeneratorTest" time="0.003"/>
  <testcase name="testBadInitFile2" classname="crypt.impl.certificate.X509V3GeneratorTest" time="0.016"/>
  <testcase name="testGetInstance" classname="crypt.impl.certificate.X509V3GeneratorTest" time="0.002"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
